suite-native/navigation/src/navigators.ts

```diff
@@ -202,9 +203,8 @@ export type DeviceSettingsStackParamList = {
         type: PinActionType;
     };
     [DeviceSettingsStackRoutes.DeviceAuthenticity]: undefined;
+    [DeviceSettingsStackRoutes.FirmwareUpdateStack]: undefined;
```

Comment:
The type of the stack has to be provided otherwise the typing won't work as expected.

```suggestion
    [DeviceSettingsStackRoutes.FirmwareUpdateStack]: NavigatorScreenParams<FirmwareUpdateStackParamList>;
```

Reply:
Good catch, thanks!